.mergify.yml

```diff
@@ -17,3 +17,17 @@ pull_request_rules:
         strict: smart
         strict_method: merge
         commit_message: title+body
+  - name: Automatically approve Dependabot PRs
+    conditions:
+      - base=master
+      - status-success=AWS CodeBuild us-west-2 (ecs-cli-v2-ci)
```

Comment:
sweet, can we also add this: `- status-success=Semantic Pull Request` just in case

Reply:
fo sho!